src/main/java/mate/academy/model/ShoppingCart.java

```diff
@@ -0,0 +1,66 @@
+package mate.academy.model;
+
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.JoinTable;
+import jakarta.persistence.MapsId;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import java.util.List;
+
+@Entity
+@Table(name = "shopping_carts")
+public class ShoppingCart {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToMany(cascade = {CascadeType.MERGE})
```

Comment:
I guess we don't need cascade type MERGE, because in Dao classes/methods you fetch relationships
```suggestion
    @OneToMany
```